### PR TITLE
Fortnum & Mason Case Study

### PR DESCRIPTION
**Pull request by mveritym**
### Motivation

As a potential customer, I want to see more information about our work with F&M, so that I can decide if Red Badger might be right for me.

### Test plan

Check the F&M case study page matches the designs on Zepplin.

### Pre-merge checklist

- [x] Documentation
- [x] Unit tests
- [x] Reviews
  - [x] Code review
  - [x] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

fortnum

### CODE REVIEW

**Comment by chris-brookes** [comment]
@mveritym - Can you also create the redirect for the F&M page, similar to the retailer one please?
 https://github.com/redbadger/website-next-proxy/pull/33/files

p.s. Can you have a look at the file conflict too? :-)

**Comment by chris-brookes** [comment]
@mveritym 
1. The background colour is missing from the main content section. (#F8F8F8 or 248, 248, 248)
2. The main header image should always touch both sides of the page and have also have a max height of 456px in desktop view (match retail case study page). As the page is stretched the image should zoom in and the top and bottom should be equally cropped off/out of view. See image 1.
3. The text and image layout is not inline with the column spacing design. There is not enough space on the right hand side. The each column spacing is too wide. See columns spacing layout and width values in image 2. (We're not using a column layout template at the moment, but you can see where the extra spacing should be from the image. You can also see this image by switching on the grid in Zeplin. The switch is by the values I've highlighted in the picture. Come over and I can go through it with you if you need a hand.)

**IMAGE 1**

screen shot 2017-04-03 at 18 02 50



**IMAGE 2**

screen shot 2017-04-03 at 18 41 40

File: site/pages/our-work/case-study/fortnums/index.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+          <span className={styles.content__redTitle}>
+            Do the right thing. Do the thing right.
+          </span>
+          Embracing organisational change and implementing cutting-edge technology
+        </h2>
+        <div className={cx('contactBox', 'showMedium')}>
+          <h2 className={styles.contactBox__heading}>
+            Project in mind?
+          </h2>
+          <a href="#contactUs" className={styles.contactBox__button}>Tell us more</a>
```
Comment (grahammendick) [inline site/pages/our-work/case-study/fortnums/index.js]: We put a workaround into `routes/index.js` to handle #contactUs in the Home page Url (we had to jump to contact us across different routes and the hash doesn't work well with client side navigation). This workaround is messing up your contact us link. To see why, try this
* Go to your page
* Click the Tell us more
* Press back

Result: the Url still has contact us hash in - but it shouldn't.

You can fix this by adding a check for the `home` key in `routes\defintions`
```
    if (hrefElement.hash === '#contactUs') {
      const { state, data } = stateNavigator.parseLink(url);
      if (state.key === 'home') {
        url = stateNavigator.getNavigationLink(state.key, { ...data, contactUs: true });
      }
    }
```

File: site/css/typography/_fonts.css [hunk 2]
```diff
@@ -152,4 +163,14 @@
     font-size: 40px;
     line-height: 1.1em;
   }
+
+  .fontM2 {
+    font-size: 40px;
+    line-height: 1.1em;
+  }
```
Comment (grahammendick) [inline site/css/typography/_fonts.css]: Changing fontM2 effects lots of other pages. Just double checking that this is correct?

File: site/pages/our-work/case-study/fortnums/index.js [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+
+const FMCaseStudy = () => (
+  <div>
+    <div className={styles.header}>
+      <div className={styles.header__container}>
+        <div className={styles.header__imageContainer}>
+          <picture>
+            <div className={styles.helper} />
+            <source srcSet={headerMedium} media="(min-width: 690px)" />
+            <source srcSet={headerLarge} media="(min-width: 980px)" />
```
Comment (grahammendick) [inline site/pages/our-work/case-study/fortnums/index.js]: Nice 👍 !

File: site/pages/our-work/case-study/fortnums/index.js (hunk 1 above)
Comment (mveritym) [inline site/pages/our-work/case-study/fortnums/index.js]: I can't reproduce this 🤔 is it browser specific or have other updates changed this?

Comment (grahammendick) [inline site/pages/our-work/case-study/fortnums/index.js]: Switching to `Link` components make it harder to recreate, but try the following (I added one extra step)

* Go your page
* **Refresh the page**
* Click the Tell us more
* Press back

Comment (mveritym) [inline site/pages/our-work/case-study/fortnums/index.js]: I still can't reproduce this 😬

**Comment by mveritym** [comment]
This is ready for another review 🐙

**Comment by chris-brookes** [comment]
The 'Tell us more' button in the contact box is not aligned correctly when the page is viewed in the smallest desktop view (980px). The button is touching the bottom of the box. The box is also too close to the edge of the page and not leaving a big enough margin.
The contact box's margins/size etc. should match the one on the other case study page https://red-badger.com/our-work/case-study/retailer

screen shot 2017-04-19 at 10 44 57

**Comment by chris-brookes** [comment]
@mveritym 
Padding and alignment issues still appearing when viewing with 980px width. See image attached.

screen shot 2017-04-19 at 14 03 08

**Comment by anurbanromantic** [comment]
@mveritym I only have a few things.
– The large images are actually gifs. The files are here > https://drive.google.com/drive/u/0/folders/0B1BSwE04MMWPNk9UWVdseGFJZVk
– Those gifs should also be smaller on Desktop. The right edges of the images should be aligned with the body text please. Tablet and mobile are fine.
– The text on the tables on Desktop should also be aligned with body text on left and right. And the box should have 50px padding around the text please. Tablet and mobile are fine.
Thank you ;)

**Comment by chris-brookes** [comment]
@grahammendick - Can you do a final review and unblock the PR please.

**Comment by chris-brookes** [comment]
Waiting for final F&M sign-off before we can push live. We can not merge to stage yet because it will block any other master live deploys.

**Comment by gregdorward** [comment]
More work going to be needed once the designs have been updated to more basic website images

**Comment by gregdorward** [comment]
Blocked until #527 is approved by F&M

**Comment by pataruco** [comment]
I am closing this PR because it's implementation has changed

**Pull request closed without merging**